ambry-cloud/src/main/java/com.github.ambry.cloud/VcrRequests.java

```diff
@@ -86,12 +81,4 @@ protected ServerErrorCode validateRequest(PartitionId partition, RequestOrRespon
   protected long getRemoteReplicaLag(Store store, long totalBytesRead) {
     return -1;
   }
-
-  @Override
-  protected Map<PartitionId, ReplicaId> createLocalPartitionToReplicaMap() {
```

Comment:
Where did this code go? The implementation defined in AmbryRequests looks like it only gets partitions on the local node, whereas this gets all partitions.

Also, shouldn't this extend AmbryServerRequests if it previously extended the old AmbryRequests (to get the request deserialization code)?

Reply:
I removed this method because localPartitionToReplicaMap is only used for disk-based partitions, e.g. checking if a disk is bad or partition is not on this node.  I don't think the VCR needs this.  @ankagrawal or @zzmao can you weigh in here?

Reply:
Ah, I see the change you made in AmbryRequests now to remove the check that used to be in `validateRequests`. It seems like the admin request support has been removed from VcrRequests as well (by having this not extend AmbryServerRequests). I think that is ok since most of those operations were not fully supported anyways (although some of them would be nice to haves in the future).

Reply:
The logic looks good.